### DISCUSSION TITLE
v1.98.6

### DISCUSSION THREAD

**Discussion by cyring** [cyring/CoreFreq#516] [Show and tell]
# [AMD]
* [V2000 Series] Adding the Ryzen Embedded V2A46
* [Family 1Ah][All Families] Refactoring topology for CCD cluster
  Confirmed `7950X`, `3950X`

# [Intel]
* ODCM is confirmed working on Raptor Lake architecture
  Confirmed `i9-14900K`

# [UI]
* Increased max ratio in HWP condition to avoid a zero frequency
  Confirmed `i9-14900K`

# [Build]
* Print other variables from Makefile recipe info
  `CORE_COUNT`
  `TASK_ORDER`
  `MAX_FREQ_HZ`
  `HWM_CHIPSET`
## [CI]
* [AArch64] Commenting out the `debian-testing` and `alpine-latest`

# [Doc]
* Mention the AMD family 1Ah support in README

----

# CoreFreq ISO
Instructions in Wiki / [LiveCD](https://github.com/cyring/CoreFreq/wiki/Live-CD)
SHA1 of the attached image
```
41492ffb91a987b34b71f021d7a4d19aca24a276
```

<hr /><em>This discussion was created from the release <a href='https://github.com/cyring/CoreFreq/releases/tag/1.98.6'>v1.98.6</a>.</em>

**Comment by cyring** [comment]
@bubbleguuum 
Another minor version just one week after. Sorry if it's too close.

**Comment by bubbleguuum** [comment]
No problem, it just takes a couple of minutes to update the package, which I just did.
